<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MultiMC 5</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/multimc/multimc.qrc">
    <normaloff>:/icons/multimc/scalable/apps/multimc.svg</normaloff>:/icons/multimc/scalable/apps/multimc.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Main Toolbar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddInstance"/>
   <addaction name="actionCopyInstance"/>
   <addaction name="separator"/>
   <addaction name="actionViewInstanceFolder"/>
   <addaction name="actionViewCentralModsFolder"/>
   <addaction name="actionRefresh"/>
   <addaction name="separator"/>
   <addaction name="actionCheckUpdate"/>
   <addaction name="actionSettings"/>
   <addaction name="separator"/>
   <addaction name="actionReportBug"/>
   <addaction name="actionAbout"/>
   <addaction name="separator"/>
   <addaction name="actionCAT"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="instanceToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>Instance Toolbar</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftToolBarArea|Qt::RightToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionChangeInstIcon"/>
   <addaction name="actionLaunchInstance"/>
   <addaction name="actionLaunchInstanceOffline"/>
   <addaction name="separator"/>
   <addaction name="actionEditInstNotes"/>
   <addaction name="actionChangeInstGroup"/>
   <addaction name="separator"/>
   <addaction name="actionInstanceSettings"/>
   <addaction name="actionChangeInstMCVersion"/>
   <addaction name="actionChangeInstLWJGLVersion"/>
   <addaction name="actionEditInstMods"/>
   <addaction name="actionViewSelectedInstFolder"/>
   <addaction name="actionConfig_Folder"/>
   <addaction name="separator"/>
   <addaction name="actionDeleteInstance"/>
  </widget>
  <widget class="QToolBar" name="newsToolBar">
   <property name="windowTitle">
    <string>News Toolbar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionMoreNews"/>
  </widget>
  <action name="actionAddInstance">
   <property name="icon">
    <iconset theme="new">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Add Instance</string>
   </property>
   <property name="toolTip">
    <string>Add a new instance.</string>
   </property>
   <property name="statusTip">
    <string>Add a new instance.</string>
   </property>
  </action>
  <action name="actionViewInstanceFolder">
   <property name="icon">
    <iconset theme="viewfolder">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>View Instance Folder</string>
   </property>
   <property name="toolTip">
    <string>Open the instance folder in a file browser.</string>
   </property>
   <property name="statusTip">
    <string>Open the instance folder in a file browser.</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset theme="refresh">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="toolTip">
    <string>Reload the instance list.</string>
   </property>
   <property name="statusTip">
    <string>Reload the instance list.</string>
   </property>
  </action>
  <action name="actionViewCentralModsFolder">
   <property name="icon">
    <iconset theme="centralmods">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>View Central Mods Folder</string>
   </property>
   <property name="toolTip">
    <string>Open the central mods folder in a file browser.</string>
   </property>
   <property name="statusTip">
    <string>Open the central mods folder in a file browser.</string>
   </property>
  </action>
  <action name="actionCheckUpdate">
   <property name="icon">
    <iconset theme="checkupdate">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Check for Updates</string>
   </property>
   <property name="toolTip">
    <string>Check for new updates for MultiMC</string>
   </property>
   <property name="statusTip">
    <string>Check for new updates for MultiMC</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset theme="settings">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Change settings.</string>
   </property>
   <property name="statusTip">
    <string>Change settings.</string>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
  </action>
  <action name="actionReportBug">
   <property name="icon">
    <iconset theme="bug">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Report a Bug</string>
   </property>
   <property name="toolTip">
    <string>Open the bug tracker to report a bug with MultiMC.</string>
   </property>
   <property name="statusTip">
    <string>Open the bug tracker to report a bug with MultiMC.</string>
   </property>
  </action>
  <action name="actionMoreNews">
   <property name="icon">
    <iconset theme="news">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>More News</string>
   </property>
   <property name="iconText">
    <string>More news...</string>
   </property>
   <property name="toolTip">
    <string>Open the MultiMC development blog to read more news about MultiMC.</string>
   </property>
   <property name="statusTip">
    <string>Open the MultiMC development blog to read more news about MultiMC.</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="about">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>About MultiMC</string>
   </property>
   <property name="toolTip">
    <string>View information about MultiMC.</string>
   </property>
   <property name="statusTip">
    <string>About MultiMC</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionLaunchInstance">
   <property name="text">
    <string>Play</string>
   </property>
   <property name="toolTip">
    <string>Launch the selected instance.</string>
   </property>
   <property name="statusTip">
    <string>Launch the selected instance.</string>
   </property>
  </action>
  <action name="actionRenameInstance">
   <property name="text">
    <string>Instance Name</string>
   </property>
   <property name="toolTip">
    <string>Rename the selected instance.</string>
   </property>
   <property name="statusTip">
    <string>Rename the selected instance.</string>
   </property>
  </action>
  <action name="actionChangeInstGroup">
   <property name="text">
    <string>Change Group</string>
   </property>
   <property name="toolTip">
    <string>Change the selected instance's group.</string>
   </property>
   <property name="statusTip">
    <string>Change the selected instance's group.</string>
   </property>
  </action>
  <action name="actionChangeInstIcon">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/instances/instances.qrc">
     <normaloff>:/icons/instances/infinity</normaloff>:/icons/instances/infinity</iconset>
   </property>
   <property name="text">
    <string>Change Icon</string>
   </property>
   <property name="toolTip">
    <string>Change the selected instance's icon.</string>
   </property>
   <property name="statusTip">
    <string>Change the selected instance's icon.</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionEditInstNotes">
   <property name="text">
    <string>Edit Notes</string>
   </property>
   <property name="toolTip">
    <string>Edit the notes for the selected instance.</string>
   </property>
   <property name="statusTip">
    <string>Edit the notes for the selected instance.</string>
   </property>
  </action>
  <action name="actionInstanceSettings">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Change settings for the selected instance.</string>
   </property>
   <property name="statusTip">
    <string>Change settings for the selected instance.</string>
   </property>
  </action>
  <action name="actionMakeDesktopShortcut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Make Shortcut</string>
   </property>
   <property name="toolTip">
    <string>Make a shortcut on the desktop for the selected instance.</string>
   </property>
   <property name="statusTip">
    <string>Make a shortcut on the desktop for the selected instance.</string>
   </property>
  </action>
  <action name="actionManageInstSaves">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Manage Saves</string>
   </property>
   <property name="toolTip">
    <string>Manage saves for the selected instance.</string>
   </property>
   <property name="statusTip">
    <string>Manage saves for the selected instance.</string>
   </property>
  </action>
  <action name="actionEditInstMods">
   <property name="text">
    <string>Edit Mods</string>
   </property>
   <property name="toolTip">
    <string>Edit the mods for the selected instance.</string>
   </property>
   <property name="statusTip">
    <string>Edit the mods for the selected instance.</string>
   </property>
  </action>
  <action name="actionChangeInstMCVersion">
   <property name="text">
    <string>Change Version</string>
   </property>
   <property name="toolTip">
    <string>Change the selected instance's Minecraft version.</string>
   </property>
   <property name="statusTip">
    <string>Change the selected instance's Minecraft version.</string>
   </property>
  </action>
  <action name="actionChangeInstLWJGLVersion">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Change LWJGL</string>
   </property>
   <property name="toolTip">
    <string>Change the version of LWJGL for the selected instance to use.</string>
   </property>
   <property name="statusTip">
    <string>Change the version of LWJGL for the selected instance to use.</string>
   </property>
  </action>
  <action name="actionViewSelectedInstFolder">
   <property name="text">
    <string>Instance Folder</string>
   </property>
   <property name="toolTip">
    <string>Open the selected instance's root folder in a file browser.</string>
   </property>
   <property name="statusTip">
    <string>Open the selected instance's root folder in a file browser.</string>
   </property>
  </action>
  <action name="actionDeleteInstance">
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete the selected instance.</string>
   </property>
   <property name="statusTip">
    <string>Delete the selected instance.</string>
   </property>
  </action>
  <action name="actionConfig_Folder">
   <property name="text">
    <string>Config Folder</string>
   </property>
   <property name="toolTip">
    <string>Open the instance's config folder</string>
   </property>
  </action>
  <action name="actionCAT">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="cat">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Meow</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;It's a fluffy kitty :3&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionCopyInstance">
   <property name="icon">
    <iconset theme="copy">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Copy Instance</string>
   </property>
   <property name="toolTip">
    <string>Copy the selected instance.</string>
   </property>
   <property name="statusTip">
    <string>Add a new instance.</string>
   </property>
  </action>
  <action name="actionManageAccounts">
   <property name="text">
    <string>Manage Accounts</string>
   </property>
   <property name="toolTip">
    <string>Manage your Mojang or Minecraft accounts.</string>
   </property>
  </action>
  <action name="actionLaunchInstanceOffline">
   <property name="text">
    <string>Play Offline</string>
   </property>
   <property name="toolTip">
    <string>Launch the selected instance in offline mode.</string>
   </property>
   <property name="statusTip">
    <string>Launch the selected instance.</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources/multimc/multimc.qrc"/>
  <include location="../resources/instances/instances.qrc"/>
 </resources>
 <connections/>
</ui>
